test/__snapshots__/cli.test.ts.snap

```diff
@@ -264,6 +264,238 @@ namespace MyPackage
 }
 `;
 
+exports[`golang output 1`] = `
+Object {
+  "package/LICENSE": "Apache License
```

Comment:
Is this consistent with other languages? Do we really need the LICENSE file here?

Reply:
As nothing Go specific was added, this should be consistent with other languages.
I did a bit of digging and it seems like this was introduced via https://github.com/aws/jsii/pull/2604 and [this line](https://github.com/cdklabs/jsii-srcmak/blob/6bcc4a610cd845f1e3492d589fe0e59e9445e50e/src/compile.ts#L60) causes the Apache License to be included.
I think we can set it to `UNLICENSED` to get rid of the generated license file. However, it is already included in [other snapshots already](https://github.com/cdklabs/jsii-srcmak/blob/1870624a3965c53e675bc31a1b99a87530a42c85/test/__snapshots__/cli.test.ts.snap#L9).

Reply:
It's odd to me that we include the license file in there. Can we remove?